frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js

```diff
@@ -32,6 +36,7 @@ class DatasetActionView extends React.PureComponent<Props, State> {
 
   clearCache = async (dataset: APIMaybeUnimportedDataset) => {
     await triggerDatasetClearCache(dataset.dataStore.url, dataset);
+    await triggerDatasetClearThumbnailCache(dataset);
```

Comment:
@fm3 Does the backend expect these requests in order? Otherwise you could use Promise.all to execute them concurrently :)

```javascript
await Promise.all([
  triggerDatasetClearCache(dataset.dataStore.url, dataset),
  triggerDatasetClearThumbnailCache(dataset),
]);
```